client/package.json

```diff
@@ -19,20 +19,24 @@
     "@angular/core": "^4.0.0",
     "@angular/forms": "^4.0.0",
     "@angular/http": "^4.0.0",
-    "@angular/material": "^2.0.0-beta.10",
     "@angular/platform-browser": "^4.0.0",
     "@angular/platform-browser-dynamic": "^4.0.0",
     "@angular/router": "^4.0.0",
-    "@types/socket.io-client": "~1.4.30",
+    "@types/socket.io-client": "~1.4.29",
+    "@types/express": "~4.0.36",
+    "express": "~4.10.0",
+    "@angular/material": "^2.0.0-beta.12",
```

Comment:
I like a lot this update. However, I think we can use even latest version of  Angular Material here: `"@angular/material": "^5.0.0-rc.2"`.
What do you think?

Reply:
As I'm quite new to ES(in all it's forms) development, I think your guess is as good (and/or better) than mine. But it seems as though you are right. I don't have access to my workstation until tomorrow, but I certainly will check it out and make any changes you request!